CMakeLists.txt

```diff
@@ -57,6 +57,12 @@ if(WIN32 AND BUILD_SHARED_LIBS)
   set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
 endif()
 
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
+set(BUILD_RPATH_USE_ORIGIN TRUE)
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+set(CMAKE_INSTALL_RPATH "$ORIGIN")
+set(CMAKE_BUILD_RPATH "$ORIGIN")
```

Comment:
I have no idea, but is this redundant with `set(BUILD_RPATH_USE_ORIGIN TRUE)`?

Reply:
From https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_RPATH.html

- `CMAKE_BUILD_RPATH` sets the `RPATH` directly to the given content.

From https://cmake.org/cmake/help/latest/prop_tgt/BUILD_RPATH_USE_ORIGIN.html

- `BUILD_RPATH_USE_ORIGIN` is set by `CMAKE_BUILD_RPATH_USE_ORIGIN`.

I am not sure what is the difference between `BUILD_RPATH_USE_ORIGIN` and 
`set(CMAKE_BUILD_RPATH "$ORIGIN")`. But the current setting causes no problem.
I would rather leave it unchanged.